cmd/podman/common/create.go

```diff
@@ -393,6 +393,11 @@ func GetCreateFlags(cf *ContainerCLIOpts) *pflag.FlagSet {
 		"rootfs", false,
 		"The first argument is not an image but the rootfs to the exploded container",
 	)
+	createFlags.StringVar(
+		&cf.SdNotifyMode,
+		"sdnotify", "container",
```

Comment:
Should probably be in containers.conf github.com/containers/common/pkg/config.  But this is fine for now.

Reply:
These should become constants not magic strings.